docs/spec/p2p/peer.md

```diff
@@ -27,27 +27,24 @@ Both handshakes have configurable timeouts (they should complete quickly).
 ### Authenticated Encryption Handshake
 
 Tendermint implements the Station-to-Station protocol
-using ED25519 keys for Diffie-Helman key-exchange and NACL SecretBox for encryption.
+using Ed25519 keys for Diffie-Helman key-exchange and chacha20poly1305 for encryption.
```

Comment:
The Montgomery keys are typically referred to as x25519

Reply:
Good catch! Thanks